tests/recipe/UPDATE_recipes.test.js

```diff
@@ -0,0 +1,89 @@
+// External libraries
+const supertest = require("supertest");
+const mongoose = require("mongoose");
+
+// Internal modules
+const app = require("../../app");
+const Recipe = require("../../models/recipe");
+const User = require("../../models/user");
+const { initialUsers, recipesInDb, initialRecipes } = require("../test_helper");
+const {
+  createUserAndLogin,
+  addRecipe,
+  updateRecipe,
+} = require("../api_test_helpers");
+
+// Constants
+const api = supertest(app);
+const STATUS_CODE = require("../../shared/errorCode");
+const INVALID_RECIPE_ID = "1234567890";
```

Comment:
why is this invalid?

Reply:
just a test? 
If the recipe-id is invalid is should not update

Reply:
but why is this specific number invalid?